oclc.py

```diff
@@ -1154,6 +1349,14 @@ def process_oclc_edition(self, original_identifier, edition):
             oclc_number.equivalent_to(
                 self.output_source, isbn_identifier, 1)
 
+        metadata = Metadata(self.output_source)
+        # Return contributor information.
+        VIAF_ID = re.compile("^http://viaf.org/viaf/([0-9]+)/$")
+        for viaf in edition['creator_viafs']:
+            viaf = VIAF_ID.search(viaf).groups()[0]
```

Comment:
There's a chance this regex won't match, because you put all viaf.org URLs into this list. With OCLC+LD this is the sort of problem that will happen one time in two million. Either use the same regex when putting URLs into this list down in the API, or skip a URL instead of processing it if search() returns None.